libs/community/langchain_community/document_loaders/url_playwright.py

```diff
@@ -135,8 +153,11 @@ def __init__(
         continue_on_failure: bool = True,
         headless: bool = True,
         remove_selectors: Optional[List[str]] = None,
+        keep_only_selectors: Optional[List[str]] = None,
         evaluator: Optional[PlaywrightEvaluator] = None,
         proxy: Optional[Dict[str, str]] = None,
+        header: Optional[str] = ('Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) '
```

Comment:
This is unrelated, correct?

Let's call this `user_agent`.

Reply:
Yes it's un related. It's done.